src/SIL.LCModel/DomainImpl/OverridesLing_Lex.cs

```diff
@@ -3297,6 +3297,16 @@ public string ReferenceName
 			}
 		}
 
+		/// <summary>
+		/// Remove one reference to the target object from one of your atomic reference properties.
+		/// </summary>
+		internal override void RemoveAReferenceCore(ICmObject target)
+		{
+			var flid = this.Cache.CustomProperties.Where(x => x.Value == target && x.Key.Item1 == this).Select(y => y.Key.Item2).FirstOrDefault();
+			if (flid > 0)
+				SetNonModelPropertyForSDA(flid, null, true);
+		}
```

Comment:
James, I think you should add one line to the end of the method:
This shouldn't change any behavior, but it makes it consistent and might avoid a future bug.
base.RemoveAReferenceCore(target);

Reply:
Jason, I added base.RemoveAReferenceCore(target).